src/rust/ide/view/graph-editor/src/component/breadcrumbs.rs

```diff
@@ -399,8 +399,8 @@ impl Breadcrumbs {
 
             // === Project Name ===
 
-            eval frp.project_name((name) model.project_name.frp.name(name));
-            frp.source.project_name <+ model.project_name.frp.output.name;
+            eval frp.input.project_name((name) model.project_name.name(name));
```

Comment:
this call `model.project_name.name(name)` looks strange, and the clash of input and output node is also not desired...

Can we rename input to `set_name` @wdanilo ?

Reply:
> Can we rename input to `set_name`
I would be for that. Especially since we have an in and output here.

Reply:
lets do it